site/content/docs/reference.md

```diff
@@ -141,6 +139,34 @@ spec:
   selector:
     app.tanzu.vmware.com/workload-type: web
 
+  # parameters to override the defaults from the templates.
+  # if a resource in the supply-chain specifies a parameter
+  # of the same name that resource parameter clobber what is
+  # specified here at the top level (this includes specification
+  # as `value` vs `default`)
+  #
```

Comment:
I wonder if this could be reworded a little differently, assuming this is true something like:

_List of parameters to override supply-chain template parameter defaults.  If a supply-chain contains a default of the same parameter name then it will be replaced.  For reference parameter values in supply-chain templates are mandatory and cannot be overridden._

Reply:
or maybe a copy / paste of the description in this PR as that seems clearer

Reply:
We still need a fairly comprehensive overhaul of our docs. To me the clearest way to describe the behaviour is simply by enumerating all the possibilities:

supply chain resource entry **value** (highest precedence)
supply chain **value**
workload **value**
supply chain resource entry **default**
supply chain **default**
template **default** (lowest precedence)

Reply:
I think the most subtle distinction (the one I have difficulty describing succinctly) is the situation where the supply chain specifies a value, but the individual resource specifies a default. In this case, if the workload specifies a value, the workload's value will be respected. That's not evident from the hierarchy represented here. That's the behavior that I'm referencing when I say the resource clobbers the supply chain specification. The resource can not only change the default, it can change a supply chain's _value_ into a _default_.

Reply:
> The resource can not only change the default, it can change a supply chain's value into a default.